docs/Developer-Guide.md

```diff
@@ -482,7 +486,48 @@ See [Set up a debug console](#set-up-a-debug-console).
 $ sudo docker info 2>/dev/null | grep -i "default runtime" | cut -d: -f2- | grep -q runc  && echo "SUCCESS" || echo "ERROR: Incorrect default Docker runtime"
 ```
 
-## Set up a debug console
+## Set up a debug console(The easy way)
+
+Kata containers 2.0 support a shell simulated *console* for quickly debug purpose. This approach use `vsock` to connect shell running inside guest started by agent. The good aspect is that we need not modify guest image or despite using what device that hypervisors support. Only `/bin/sh` or `/bin/bash` are necessary.
+
+### Enable agent debug console
+
+Change your `configuration.toml`, add agent debug parameters.
+
+```
+kernel_params = "agent.debug_console agent.debug_console_vport=1026"
```

Comment:
Can we make it internal so that users are not aware of which vsock port the debug console is using? For example:
1. add a new option `enable_debug_console`
2. when enabled, append `agent.debug_console agent.debug_console_vport=1026` to guest kernel command line automatically

Reply:
Yes, I think we can add `enable_debug_console` to `agent` section of Kata's TOML configuration file.